### ISSUE TITLE
[Bug]:  V1 1020

### ISSUE THREAD

**Issue by Liu-Angelo** [acheong08/ChatGPT#1119]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

<body>
   <div class="cf-main-wrapper" role="main">
      <div class="cf-header cf-section">
         <div class="cf-error-title">
            <h1>Access denied</h1>
            <span class="cf-code-label">Error code <span>1020</span></span>
         </div>
         <div class="cf-error-description">
            <p>You do not have access to bypass.duti.tech.</p><p>The site owner may have set restrictions that prevent you from accessing the site.</p>
         </div>
      </div>
   </div>

   <div class="cf-details-wrapper">
      <div class="cf-section" role="region">
         <div class="cf-expandable" id="cf-details-wrapper-expandable">
            <label for="error-details-checkbox" title="Error details" class="cf-expandable-btn">
               <p class="cf-dropdown-title">Error details</p>
               <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgBAMAAACBVGfHAAAAElBMVEUAAAAwMDAxMTEyMjIwMDAxMTF+89HTAAAABXRSTlMAf2CAMKS61bwAAABTSURBVCjPzcq7DcAgFENR5zMATfo06TNCViAS+6+CeFi6gglw4eLqaPVtaQpXnkApaQT/k0dw70EAUhCA1AnABGACMAGYAEwAkCOAydv+I5xaZhXWbQrD80TkkQAAAABJRU5ErkJggg=="
               class="cf-caret-icon" id="caret-icon" alt="Caret icon" />
            </label>

            <input id="error-details-checkbox" class="hidden" type="checkbox">

            <div class="cf-expandable-error-info hidden">
               <p class="cf-error-copy-description">Provide the site owner this information.</p>
               <button class="cf-click-to-copy-btn" id="click-to-copy-btn" title="Click to copy" type="button">
<div class="cf-error-wrapper" id="error-details"><p class="cf-error-details-endpoint">I got an error when visiting bypass.duti.tech/api/conversation.</p>

### Steps to reproduce the problem

<body>
   <div class="cf-main-wrapper" role="main">
      <div class="cf-header cf-section">
         <div class="cf-error-title">
            <h1>Access denied</h1>
            <span class="cf-code-label">Error code <span>1020</span></span>
         </div>
         <div class="cf-error-description">
            <p>You do not have access to bypass.duti.tech.</p><p>The site owner may have set restrictions that prevent you from accessing the site.</p>
         </div>
      </div>
   </div>

   <div class="cf-details-wrapper">
      <div class="cf-section" role="region">
         <div class="cf-expandable" id="cf-details-wrapper-expandable">
            <label for="error-details-checkbox" title="Error details" class="cf-expandable-btn">
               <p class="cf-dropdown-title">Error details</p>
               <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgBAMAAACBVGfHAAAAElBMVEUAAAAwMDAxMTEyMjIwMDAxMTF+89HTAAAABXRSTlMAf2CAMKS61bwAAABTSURBVCjPzcq7DcAgFENR5zMATfo06TNCViAS+6+CeFi6gglw4eLqaPVtaQpXnkApaQT/k0dw70EAUhCA1AnABGACMAGYAEwAkCOAydv+I5xaZhXWbQrD80TkkQAAAABJRU5ErkJggg=="
               class="cf-caret-icon" id="caret-icon" alt="Caret icon" />
            </label>

            <input id="error-details-checkbox" class="hidden" type="checkbox">

            <div class="cf-expandable-error-info hidden">
               <p class="cf-error-copy-description">Provide the site owner this information.</p>
               <button class="cf-click-to-copy-btn" id="click-to-copy-btn" title="Click to copy" type="button">
<div class="cf-error-wrapper" id="error-details"><p class="cf-error-details-endpoint">I got an error when visiting bypass.duti.tech/api/conversation.</p>

### What should have happened?

<body>
   <div class="cf-main-wrapper" role="main">
      <div class="cf-header cf-section">
         <div class="cf-error-title">
            <h1>Access denied</h1>
            <span class="cf-code-label">Error code <span>1020</span></span>
         </div>
         <div class="cf-error-description">
            <p>You do not have access to bypass.duti.tech.</p><p>The site owner may have set restrictions that prevent you from accessing the site.</p>
         </div>
      </div>
   </div>

   <div class="cf-details-wrapper">
      <div class="cf-section" role="region">
         <div class="cf-expandable" id="cf-details-wrapper-expandable">
            <label for="error-details-checkbox" title="Error details" class="cf-expandable-btn">
               <p class="cf-dropdown-title">Error details</p>
               <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgBAMAAACBVGfHAAAAElBMVEUAAAAwMDAxMTEyMjIwMDAxMTF+89HTAAAABXRSTlMAf2CAMKS61bwAAABTSURBVCjPzcq7DcAgFENR5zMATfo06TNCViAS+6+CeFi6gglw4eLqaPVtaQpXnkApaQT/k0dw70EAUhCA1AnABGACMAGYAEwAkCOAydv+I5xaZhXWbQrD80TkkQAAAABJRU5ErkJggg=="
               class="cf-caret-icon" id="caret-icon" alt="Caret icon" />
            </label>

            <input id="error-details-checkbox" class="hidden" type="checkbox">

            <div class="cf-expandable-error-info hidden">
               <p class="cf-error-copy-description">Provide the site owner this information.</p>
               <button class="cf-click-to-copy-btn" id="click-to-copy-btn" title="Click to copy" type="button">
<div class="cf-error-wrapper" id="error-details"><p class="cf-error-details-endpoint">I got an error when visiting bypass.duti.tech/api/conversation.</p>

### Version where the problem happens

Name: revChatGPT
Version: 3.3.1
Summary: ChatGPT is a reverse engineering of OpenAI's ChatGPT API
Home-page: https://github.com/acheong08/ChatGPT
Author: Antonio Cheong
Author-email: acheong@student.dalat.org
License: GNU General Public License v2.0
Location: /usr/local/python3/lib/python3.10/site-packages
Requires: httpx, OpenAIAuth, prompt-toolkit, requests, tiktoken
Required-by: 

### What Python version are you running this with?

Python 3.10.5

### Command Line Arguments

```Shell
chatbot = Chatbot(config={
    "email": "*****1226@aliyun.com",
    "password": "****8123",
    "paid": True
})
```

### Console logs

```Shell
<body>
   <div class="cf-main-wrapper" role="main">
      <div class="cf-header cf-section">
         <div class="cf-error-title">
            <h1>Access denied</h1>
            <span class="cf-code-label">Error code <span>1020</span></span>
         </div>
         <div class="cf-error-description">
            <p>You do not have access to bypass.duti.tech.</p><p>The site owner may have set restrictions that prevent you from accessing the site.</p>
         </div>
      </div>
   </div>
```

**Comment by ymybxx** [comment]
需要支持绕过cloudFlare。。

**Comment by acheong08** [comment]
Duplicate of #1118
